docs/Style-Guide.md

```diff
@@ -0,0 +1,180 @@
+
+**Example**
+
+Less Like this: we can use the following plugins.
+
+More like this: You can use the following plugins.
+
+> According to [Wikipedia](#), ***You*** is usually a second person pronoun. Also, used to refer to an indeterminate person, as a more common alternative to a very formal indefinite pronoun.
+
+## When to avoid second person "you" as the pronoun
+
+One of the main rules of formal writing such as reference documentation, or API documentation, is to avoid second person ("you"), avoiding directly addressing the reader.
+
+**Example**
+
+Less Like this: You can use the following recommendation as an example.
+
+More like this: As an example, the following recommendations should be referenced.
+
+To view a live example, refer to the [Decorators.md](/docs/Decorators.md) reference document. 
+
+
+### Avoid using condescending terms
+
+Condescending terms are words that include:
+
+* Just 
+* Easy
+* Simply
+* Basically
+* Obviously
+
+The reader may not find it easy to use the Fastify's framework and plugins, so avoid words that make it sound simple or easy. Not everyone who reads the documentation has the same level of understanding.
+
+
+### Starting with a verb
+
+Mostly start your description with a verb, which makes it simple and precise for the reader to follow. Prefer usage present tense because it's easier to read and understand than the past or future tense.
+
+**Example**
+
+ Less like this: There is a need for Nodejs to be installed before you can be able to use Fastify.
+
+ More like this: Install Nodejs to make use of Fastify.
+
+### Grammatical moods 
+
+Grammatical moods is a great way to express your writing. Avoid sounding too bossy while making a direct statement. Know when to switch between indicative, imperative, and subjunctive moods.
+
+
+**Indicative** - Use when making a factual statement or question. 
+
+Example: Since there is no testing framework available, "Fastify recommends ways to write tests".
+
+**Imperative** - Use when giving instructions, actions, commands, or when you write your headings.
+
+Example: Install dependencies before starting development.
+
+
+**Subjunctive** -  Use when making suggestions, hypothesis or non-factual statements.
+
+Example: Reading the documentation on our website is recommended to get comprehensive knowledge of the framework.
+
+### Use **active** voice instead of **passive**
+
+Using active voice is a more compact and direct way of conveying your documentation.
+
+**Example**
+
+Passive: The dependencies of the npm package is installed by nodeJs.
+
+Active: NodeJs installs the npm package dependencies.
+
+## Writing Style
+
+### Documentation titles
+
+When creating a new guide, API or reference in the `/docs/` directory, use short titles that best describe the topic of your documentation. 
+Separate each sentence with the dash *(-)* symbol, which makes your `.md` files clear and readable.
```

Comment:
"Documentation file names should be [kebab cased](https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841)..."

A file name is not a sentence.